test/index.js

```diff
@@ -813,3 +816,9 @@ test('contramap', function() {
 
   eq(Z.contramap(length, inc)('abc'), 4);
 });
+
+test('errors', function() {
+  throws(function() { Z.chainRec(Maybe); },
+         TypeError,
+         'ChainRec.methods.chainRec(...) is not a function');
+});
```

Comment:
I don't, I'm sorry.

I don't think a regression test is crucial, though, since we're replacing one "unexpected" exception with another.